src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -3709,10 +3703,8 @@ void ride_construction_toolupdate_entrance_exit(ScreenCoordsXY screenCoords)
     gMapSelectType = MAP_SELECT_TYPE_FULL;
     gMapSelectPositionA = entranceOrExitCoords;
     gMapSelectPositionB = entranceOrExitCoords;
+    gMapSelectArrowPosition = CoordsXYZ{ entranceOrExitCoords.x, entranceOrExitCoords.y, entranceOrExitCoords.z };
```

Comment:
Can't this just assign from a CoordsXYZD to a CoordsXYZ?

Reply:
It can and it now is.